### ISSUE TITLE
Feature request: Cloud SQL

### ISSUE THREAD

**Issue by jlestel** [Cyclenerd/google-cloud-pricing-cost-calculator#110]
**Is your feature request related to a problem? Please describe.**
I would like to add Cloud SQL to my encryptions.

Does this feature seem complex to you?

Any advice to give me before I look to make you a PR about this?

**Comment by Cyclenerd** [comment]
Good idea. In this [README](https://github.com/Cyclenerd/google-cloud-pricing-cost-calculator/tree/master/build) I describe how the costs are calculated. Cloud SQL SKUs are already downloaded: https://github.com/Cyclenerd/google-cloud-pricing-cost-calculator/blob/master/build/skus.sh#L23

You need to create a name for each combination. Like the Google calculator. Example: `db-standard-4`. The costs can then be calculated for the possible combinations.
